test/cases/enumBoolean.ts

```diff
@@ -0,0 +1,23 @@
+export var schema = {
+  "title": "EnumBoolean",
+  "type": "object",
+  "properties": {
+      "foo": {
+        "type" : "boolean",
```

Comment:
Maybe we could have another test _without_ a `"type": "boolean"`. Like here: https://github.com/webpack/webpack/blob/master/schemas/webpackOptionsSchema.json#L638